.github/workflows/docker.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
```

Comment:
Not needed, as fetch depth 0 already fetches tags

Reply:
alright, I wanna keep it though :3